File: BulbaCourses/BulbaCourses.PracticalMaterialsTests.Logic/Models/Tests/MTest_MainInfo.cs
using BulbaCourses.PracticalMaterialsTests.Logic.Models.Questions;
using System.Collections.Generic;

namespace BulbaCourses.PracticalMaterialsTests.Logic.Models.Tests
{
    public class MTest_MainInfo
    {
        public int Id { get; set; }

        public string Name { get; set; }        

        public ICollection<MQuestion_ChoosingAnswerFromList> Questions_ChoosingAnswerFromList { get; set; }

        public ICollection<MQuestion_SetIntoMissingElements> Questions_SetIntoMissingElements { get; set; }

        public ICollection<MQuestion_SetOrder> Questions_SetOrder { get; set; }
    }
}

File: bulbaCoursesClient/src/app/PracticalMaterialsTests/models/test/Test/MTest_MainInfo.ts
import { MQuestion_ChoosingAnswerFromList } from '../Questions/MQuestion_ChoosingAnswerFromList';
import { MQuestion_SetOrder } from '../Questions/MQuestion_SetOrder';

export interface MTest_MainInfo {
  Id:                               number;
  Name:                             string;
  Questions_ChoosingAnswerFromList: MQuestion_ChoosingAnswerFromList[];
  Questions_SetOrder:               MQuestion_SetOrder[];
}
